src/main/java/emissary/server/api/Agents.java

```diff
@@ -41,6 +45,18 @@ public Response agents() {
         return Response.ok().entity(lookupAgents()).build();
     }
 
+    @GET
+    @Path("/agents/log")
+    @Produces(MediaType.APPLICATION_JSON)
+    public Response agentsLog() {
+        AgentsResponseEntity entity = lookupAgents();
+        AgentsFormatter formatter = AgentsFormatter.builder().withHost(entity.getLocal().getHost()).build();
+        StringJoiner joiner = new StringJoiner("\n", "", "\n");
+        entity.getLocal().getAgents().forEach(agent -> joiner.add("{\"agents\":" + formatter.json(agent) + "}"));
+        entity.getErrors().forEach(err -> joiner.add("{\"agents\":" + formatter.json("error", err) + "}"));
+        return Response.ok().entity(joiner.toString()).build();
+    }
+
     @GET
     @Path("/cluster/agents")
     @Produces(MediaType.APPLICATION_JSON)
```

Comment:
I think we might want to put `@Deprecated` on this

Reply:
Added `@Deprecated` on the old methods. I labeled the new methods as `v2` to distinguish them from the older ones. Let me know if you want it named something else. I didn't want to name it `agents/ng` or something like that. Eventually we'll just remove the v1 methods and use the v2 ones.

Reply:
We don't have a /v1/agents/log so I don't know we need a /v2/agents/log for distinction.

seems like:
/agents (old style which calls /v1/agents)
/agents/log (which is new and can just be what's in /v2/agents/log)
/v1/agents
/v2/agents

is all we need?